lib/ui/screens/timer/timer_screen.dart

```diff
@@ -46,6 +46,7 @@ class _TimerScreenState extends State<TimerScreen> {
           ),
           const SizedBox(height: 80.0,),
           FloatingActionButton(
+            heroTag: 'start',
```

Comment:
Der heroTag kann entweder 'start', 'pause' oder 'stop' sein in diesem Screen. Kannst du einfach mal beliebig austauschen und gucken was dann beim Navigieren in den RunningTimerScreen passiert ;)